### ISSUE TITLE
Laravel Lumen Support

### ISSUE THREAD

**Issue by trupedia** [BKWLD/laravel-haml#17]
@weotch just tried to use the project with [Lumen](http://lumen.laravel.com/) and the composer require works, because Lumen uses most of the L5 illuminate components. But registering the ServiceProvider of the package failed, because of the missing version constant in the `Laravel\Lumen\Application`.

What are your thoughts on this, should we enhance the package to support Lumen as well? Then we have one package to rule them all...

**Comment by weotch** [comment]
Yeah, sounds nice to support Lumen too.  Yeah, maybe make the `->version()` default to 5 if that constant doesn't exist.  You down to do that?

**Comment by trupedia** [comment]
If you don't mind, I'll try it out in another branch called "lumen", if the constant / version method is the only thing that needs to be adjusted. If it'll work we can merge it into master.

**Comment by weotch** [comment]
Noice

**Comment by trupedia** [comment]
I tried it yesterday, but the issue is more complicated than only the version number. It's also an issue with the EngineResolver and registering the extensions.
I have another look over the weekend to solve this.
